safe-global/web-core
# Fix: app iframe permissions by domain

## What it solves

Resolves #1407

## How this PR fixes it
The permissions were saved for the entire domain but checked for a specific path.

It's now checking the permissions also by domain.

## How to test it
* Open the WC app
* Allow camera
* Check that the iframe has an `allow="camera"` attribute.
